lib/codecs/compressedCborld.js

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export class CompressedCborldCodec {
```

Comment:
Convention is to have these files named by their FullClassName.js which applies to the error and other classes.

Reply:
Fixed in aefb664a86f6c1c061f1c22ef11ceeddb8f3c1ee.